crates/chain/src/tx_graph.rs

```diff
@@ -182,7 +182,7 @@ pub enum CalculateFeeError {
     /// Missing `TxOut` for one or more of the inputs of the tx
     MissingTxOut(Vec<OutPoint>),
     /// When the transaction is invalid according to the graph it has a negative fee
-    NegativeFee(i64),
+    NegativeFee(SignedAmount),
```

Comment:
The `Display` impl for both `Amount` and `SignedAmount` will be denominated in BTC. Is that what we want?

https://docs.rs/bitcoin/0.31.2/bitcoin/amount/struct.SignedAmount.html#impl-Display-for-SignedAmount

Reply:
That's a good catch, thanks!

Although bitcoin::Amount is generally sats, they are using the Denomination::Bitcoin on Display, I don't think that's what we want (at least not what I had in mind).